pkgs/stdenv/darwin/default.nix

```diff
@@ -303,16 +323,7 @@ in rec {
     extraPreHook = ''
       export PATH_LOCALE=${pkgs.darwin.locale}/share/locale
     '';
-    overrides = lib.composeExtensions persistent (self: super: {
-      # Hack to make sure we don't link ncurses in bootstrap tools. The proper
-      # solution is to avoid passing -L/nix-store/...-bootstrap-tools/lib,
-      # quite a sledgehammer just to get the C runtime.
-      gettext = super.gettext.overrideAttrs (old: {
```

Comment:
@Ericson2314 why was this composed like this?

Reply:
You mean not part of `persistent`? I gather to separate concerns: we aren't "persisting" anything with this override.

Reply:
Did I do this? I kind of like it this way, not that I figure out what's going on :)